bucoffea/monojet/definitions.py

```diff
@@ -402,7 +430,86 @@ def setup_candidates(df, cfg):
     met_pt = df[f'{met_branch}_pt{jes_suffix_met}']
     met_phi = df[f'{met_branch}_phi{jes_suffix_met}']
 
-    return met_pt, met_phi, ak4, bjets, ak8, muons, electrons, taus, photons
+    # JES/JER variations in MET
+    met_pt_jerup = df[f'{met_branch}_pt_jerUp']
```

Comment:
Can we replace this part by doing:

```python
met = JaggedCandidateArray.candidatesfromcounts(
		np.ones(df.size),
		pt = ...,
		pt_jesup = ...
		)
```

?

Reply:
Sure, actually this would be way more compact. I will work on this.

Reply:
Done in two commits:
https://github.com/alpakpinar/bucoffea/commit/105339cb9f15a2a2c635b621e5e0bb585cf3c0c6
https://github.com/alpakpinar/bucoffea/commit/0582cb599eade19f2ce7a5d82cd5783707995e0d